autoapi/templates/python/data.rst

```diff
@@ -24,8 +27,7 @@
             {%- else -%}
                 {{ obj.value|string|truncate(100) }}
```

Comment:
How about add `'` for strings? Looks like it can be done here, but I am not sure how.

However, if add something like this, what to do with strings which contain `'`? For example, `"some s'tring"` will convert to `'some s'tring'`. (But this project is for *humans*, not *robots*, so should someone care about it?)

Reply:
If i recall correctly, just using the `:value:` directive should automatically detect it is a string and format it as such

Reply:
For this file:
```py
INT = 1_23
STRING = "1'23"
BIG_STRING = """
1'23
"""
DICT = {"123": 123}
LIST = [123, "1'23"]
TRUE = True
FALSE = False
```

I got with the same with patch, and without it:
![image](https://user-images.githubusercontent.com/68118654/199506690-3167eed6-c798-4d76-b431-94d0be35e7bc.png)
Also, in `LIST`, I got `"123"` (source code) → `'123'` (render) and `"1'23"` → `"1'23"`, so possibly it's a bug inside Sphinx.

Reply:
Sorry for taking so long to answer, but been busy with exams.

Created a new commit to hopefully improve multi line docs. Mind having a look?

Reply:
Well, it's better, but I didn't talk about multiline strings
![image](https://user-images.githubusercontent.com/68118654/201474211-cf328948-fef3-4c6e-8a08-85ef34aee6b9.png)

I mean, we should add `'` to value, if it's a string. Like, instead of rendering `some`, isn't better to render `'some'`?

Reply:
Oh, I missed the string type you were mentioning. I saw the multiline expanded and my mind immediately went to it. I'll have a look later!

Reply:
Should there now :)

Also fixed a lil bug with the multi-line one

Reply:
Yes, it works! Thanks!

![image](https://user-images.githubusercontent.com/68118654/201480446-0a12d1fc-b75e-4f9e-8076-07ef82318da2.png)

Only one thing to mention, if this PR no longer do the *one* thing, shouldn't it be split on few PRs?

Reply:
(and how about test such case as `"so\"me"`?)

Reply:
> Only one thing to mention, if this PR no longer do the one thing, shouldn't it be split on few PRs?

I dont really see a need, its not that far fetched, but I can split it if needed @AWhetter

> (and how about test such case as "so\"me"?)

Will add!